content/overview/_index.md

```diff
@@ -0,0 +1,26 @@
++++
+title = "Overview"
+weight = 2
+chapter = false
++++
+
+
+## Core Platform Documentation
+
+Welcome to the [Core Platform](https://www.cecg.io/core-platform/) Documentation!
+
+Core Platform is your ultimate all-in-one developer platform designed to turbocharge your software development journey from Day 1.
+
+These docs assume you've already deployed a full set of Core Platform Environments. If you haven't 
+please go to [Core Platform](https://www.cecg.io/contact-us/) to get in touch with us to set up
+your environments.
+
+## Getting started
+
+- Configure [Core Platform CLI](../corectl).
+
+- Learn more about how you can [Deploy Applications on Core Platform](../guides/deployapp/).
```

Comment:
When you onboard or create an app, there are two ways: manually or with corectl. I think the first thing tenants should do is configure corectl because, even if they don't use it during onboarding or app creation, they will need it for other guides, such as connecting to environments, listing environments, etc. Onboarding is included as a prerequisite in (../guides/deployapp/). However, I agree that we need to add onboarding tenancy to the 'Getting Started' section. Here is a 'Getting Started' list I am thinking about:

Configure corectl
Onboard tenancy
Create an application